coralogix/resource_coralogix_archive_logs.go

```diff
@@ -172,7 +172,8 @@ func flattenArchiveLogs(target *cxsdk.Target) *ArchiveLogsResourceModel {
 	}
 
 	return &ArchiveLogsResourceModel{
-		ID:                types.StringValue(""),
+		// Use non-empty ID string, as using empty string causes problems when this provider is used in Pulumi via https://github.com/pulumi/pulumi-terraform-provider
+		ID:                types.StringValue("archive-logs-settings"),
```

Comment:
@stazz the id is already computed in 206, I would pass it to the function, and then add it to the ID here: https://github.com/coralogix/terraform-provider-coralogix/blob/7cfbee54b84717703d2b7b6f05fc6183390b8ab7/coralogix/resource_coralogix_archive_logs.go#L206C1-L206C30

Reply:
Based on recent comment from @celaus , I will resolve this 👍